lib/request.js

```diff
@@ -193,34 +193,46 @@ function Request (options) {
     addAbortSignal(signal, this)
   }
 
-  return this
-}
+  {
+    const payload = this._lightMyRequest.payload
+    if (payload && payload._readableState) { // does not quack like a stream
```

Comment:
I think you mean "quacks like a stream" here :)

Reply:
I would just fix this comment.